web/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "pre-build": "rm -rf ./dist",
-    "build": "npm run pre-build && NODE_ENV=production npm run ts-node ./node_modules/.bin/webpack",
-    "build-dev": "npm run pre-build && npm run ts-node-transpile ./node_modules/.bin/webpack",
-    "watch": "npm run pre-build && nodemon --watch 'src/**' --ext 'js,jsx,ts,tsx,json' --ignore '*.spec.*' --exec npm run ts-node-transpile ./node_modules/.bin/webpack",
+    "clean": "rm -rf ./dist",
+    "build": "npm run clean && NODE_ENV=production npm run ts-node ./node_modules/.bin/webpack",
+    "build-dev": "npm run clean && npm run ts-node-transpile ./node_modules/.bin/webpack",
+    "start": "yarn ts-node node_modules/.bin/webpack serve",
```

Comment:
is it specific to yarn, or we can have the same with npm? Just for consistency between scripts ... (we could also move everything to yarn if we want)

Reply:
I just forgot that one after testings. Thanks !